src/gadgets/tables/aux.rs

```diff
@@ -275,20 +275,298 @@ impl<const REG_COUNT: usize> From<&trace::Instruction>
                     ..ch
                 },
             },
+            trace::Instruction::Or(Or { ri, rj, .. }) => Self {
+                a: SelectionA::A,
+                b: SelectionB::Reg(rj),
+                c: SelectionC::RegN(ri),
+                d: SelectionD::Unset,
```

Comment:
Is there a difference between "unset" and just setting it to zero. I did note that the paper sometimes uses 0 and sometimes use `/`, but I have just been equating them.

Reply:
I'm not sure. I made them separate in case it matters. After we have all the constraints we can try replacing `Unset` with `Zero`.